cmd/deploy/deploy_test.go

```diff
@@ -306,7 +305,7 @@ func TestCreateConfigMapWithBuildError(t *testing.T) {
 	err := c.RunDeploy(ctx, opts)
 
 	// we should get a build error because Dockerfile does not exist
-	assert.Error(t, err)
+	assert.True(t, strings.Contains(err.Error(), oktetoErrors.InvalidDockerfile))
```

Comment:
Can we use `assert.ErrorIs`?

Reply:
I think that `oktetoErrors.InvalidDockerfile` is not properly wrapped so that method doesn't work :/